app/Kconfig

```diff
@@ -389,6 +389,23 @@ config ZMK_EXT_POWER
 	bool "Enable support to control external power output"
 	default y
 
+choice ZMK_BATTERY_FETCH
+	prompt "Battery sensor fetch mode"
+
+config ZMK_BATTERY_FETCH_ALL
+	bool "Use sensor_sample_fetch()"
```

Comment:
Do we really need labels on these? Won't these mostly be in `<board>_defconfig` files?

Reply:
It appears that options in a `choice` require a prompt. I guess I could turn this into a single `bool` and assume that these are the only two options that will ever exist though.

Reply:
Keeping this as a `choice` as discussed on Discord. Using a single `bool` would let us hide the option from menuconfig, but then it's difficult to find a name that accurately describes the behavior both when it's on and off.